src/models/m_rsc.erl

```diff
@@ -215,7 +216,7 @@ get_visible(RId, Context) ->
     end.
 
 %% @doc Read a whole resource
--spec get(resource(), #context{}) -> list() | undefined.
+-spec get(resource(), #context{}) -> props() | undefined.
```

Comment:
The spec in line 219 uses a record, please define a type for the record and use that instead.